crypto-intermediary/bob.js

```diff
@@ -0,0 +1,42 @@
+console.clear();
+require("dotenv").config();
+const {
+  AccountId,
+  TokenCreateTransaction,
+  TokenType,
+  PrivateKey,
+  Client,
+  Transaction,
+  TransactionId, // <--- New Line
+} = require("@hashgraph/sdk");
+
+// Grab the OPERATOR_ID and OPERATOR_KEY from the .env file
+const BobAccountId = AccountId.fromString(process.env.BOB_OPERATOR_ID);
```

Comment:
The method to import  private keys from `.env` is different in the latest version of the SDK. Update the method here once the SDK version is updated to the latest.

`PrivateKey.fromStringED25519()`